tests/modules/elmo_test.py

```diff
@@ -131,6 +133,18 @@ def test_elmo(self):
         assert list(elmo_representations[1].size()) == [2, 7, 32]
         assert list(mask.size()) == [2, 7]
 
+    def test_elmo_4D_input(self):
```

Comment:
Can you change this test to use an expanded input with a size other than 1? That would be more robust.

An even better test would be to check that spitting up the tensor into 3D tensors and running it through elmo results in the same thing as the 4D version.

Reply:
Updated the test to run both 3D and 4D versions and compare.